src/main/java/com/gmail/uramen66/ordermovies/service/TimesLotService.java

```diff
@@ -2,37 +2,52 @@
 
 import com.gmail.uramen66.ordermovies.dto.timesLot.TimesLotDTO;
 import com.gmail.uramen66.ordermovies.dto.timesLot.TimesLotMapper;
+import com.gmail.uramen66.ordermovies.exception.ResourceNotFoundException;
 import com.gmail.uramen66.ordermovies.model.TimesLot;
 import com.gmail.uramen66.ordermovies.repositories.TimesLotRepository;
 import lombok.AllArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.stereotype.Service;
 
 @Service
 @AllArgsConstructor
 public class TimesLotService {
-    private TimesLotRepository timesLotRepository;
-    private TimesLotMapper timesLotMapper;
+    private final TimesLotRepository timesLotRepository;
+    private final TimesLotMapper timesLotMapper;
 
     public TimesLotDTO createTimesLot (TimesLotDTO timesLotDTO){
         TimesLot timesLot = TimesLot.builder()
 //                .startTime(timesLotDTO.getStartTime())
```

Comment:
is it expected? Why did it comment?

Reply:
I comment it becous it's not work. I later changed the variable type from Timestamp to Date